docs/developers.md

```diff
@@ -10,11 +10,12 @@ In order to create an APB, you will need to start with a skeleton APB directory
 
 You may create the directory structure yourself, or you can use the `apb init` command to create a simple skeleton structure, and modify it to your needs.  You will need to specify the name of your APB as a minimum input.  Visit the [APB Tooling README](https://github.com/fusor/ansible-playbook-bundle/blob/master/src/README.md) for more information.
 
-Running the `apb init` will create a APB directory structure shown below:
+Run the `apb init` command
 ```bash
 $ apb init my-apb
-
-$ tree my-apb
+```
+The following APB directory structure will be created:
+```bash
```

Comment:
when the `apb init my-apb` is ran, the folder structure is as shown.  It's also documented in the [getting started ](https://github.com/fusor/ansible-playbook-bundle/blob/master/docs/getting_started.md#creating-your-first-apb)